members/B000444.yaml

```diff
@@ -0,0 +1,106 @@
+bioguide: B000444
+contact_form:
+  method: post
+  action: ""
+  steps:
+    - visit: https://www.whitehouse.gov/contact/organization
+    - fill_in:
+        - name: submitted[first_name]
```

Comment:
Just FYI, the name field here isn't for the name of the element, it's just for a unique id on the field. EFF hasn't updated the docs, but a lot of elements don't have names so it wouldn't be reliable.